pkg/sql/delegate/show_tables.go

```diff
@@ -62,6 +62,7 @@ SELECT ns.nspname AS schema_name,
        END AS type,
        rl.rolname AS owner,
        s.estimated_row_count AS estimated_row_count,
+       s.updated_at AS estimate_update_time,
```

Comment:
nit: i think `estimate_updated_at` would be a little bit more in line with the naming convention of updatesAt columns